main.go

```diff
@@ -36,6 +36,13 @@ func main() {
 func (c *UsbPlugin) Run(cliConnection plugin.CliConnection, args []string) {
 	c.argLength = len(args)
 
+	if c.argLength < 2 {
+		if c.argLength > 0 && strings.HasPrefix(args[0], "CLI-MESSAGE-") {
+			// Internal CLI command (e.g. uninstall); don't show help text
+		}
+		return
+	}
```

Comment:
This chunk basically means any command without arguments is just busted.

Reply:
Will check. Guess the condition has to check the command as well.
I better check the backup/restore plugin also.

Reply:
Backup/restore looks good. Decided to snarf its code for here.
Done with commit d7c7b547d64d1b1bb5bd5302b23a4dc6c9bc577c